code/datums/gamemode/dynamic/dynamic_rulesets.dm

```diff
@@ -107,6 +107,8 @@
 			continue
 
 		to_chat(M, "[logo ? "[bicon(logo_icon)]" : ""]<span class='recruit'>The mode is looking for volunteers to become [initial(role_category.id)]. (<a href='?src=\ref[src];signup=\ref[M]'>Apply now!</a>)</span>[logo ? "[bicon(logo_icon)]" : ""]")
+		var/client/C = M.client
```

Comment:
Is it really needed to cast this for a single use
`C` is even less informative than `M.client` too